Control-Tower-For-Horizon/documentation/implementation-guide.md

```diff
@@ -0,0 +1,191 @@
+# Implementation Guide 
```

Comment:
There is a LOT in this README. Consider making a table of contents. Checkout https://ecotrust-canada.github.io/markdown-toc/ to help with that.